srcpkgs/vtwm/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'vtwm'
+pkgname=vtwm
+version=5.5.0
+revision=1
+build_style=gnu-configure
+configure_args="--disable-sound"
+hostmakedepends="pkg-config m4 flex"
+makedepends="libXmu-devel libXpm-devel"
+short_desc="Virtual tab window manager"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+license="X11"
+#changelog="http://vtwm.sourceforge.net/vtwm-changelog.html"
```

Comment:
changelog is now an accepted field

Reply:
Okay, I'll fix that. I remember you saying you prefer it to be right above or right below homepage?